libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -238,6 +252,15 @@ void AP_Camera::init()
         case CameraType::SCRIPTING:
             _backends[instance] = NEW_NOTHROW AP_Camera_Scripting(*this, _params[instance], instance);
             break;
+#endif
+#if AP_CAMERA_RUNCAM_ENABLED
+        // check for Scripting driver
+        case CameraType::RUNCAM:
+            _backends[instance] = NEW_NOTHROW AP_RunCam(*this, _params[instance], instance);
+            _backend_var_info[instance] = AP_RunCam::var_info;
+            AP_Param::load_object_from_eeprom(_backends[instance], _backend_var_info[instance]);
+            AP_Param::invalidate_count();
```

Comment:
```suggestion
```

done in the `load_object_from_eeprom` call

Reply:
Fixed